test/integration/suites/upstream-authority-cert-manager/03-verify-ca

```diff
@@ -13,10 +13,18 @@ expLeafURI="URI:spiffe://example.org/ns/foo/sa/bar"
 
 log-debug "verifying CA..."
 
-./bin/kubectl exec -n spire $(./bin/kubectl get pod -n spire -o name) -- /opt/spire/bin/spire-server x509 mint -spiffeID spiffe://example.org/ns/foo/sa/bar -write .
-leafURIResult=$(./bin/kubectl exec -n spire $(./bin/kubectl get pod -n spire -o name) -- cat svid.pem | openssl x509 -noout -text | grep URI | sed 's/^ *//g')
-leafIssuerResult=$(./bin/kubectl exec -n spire $(./bin/kubectl get pod -n spire -o name) -- cat svid.pem | openssl x509 -noout -issuer)
-caSubjectResult=$(./bin/kubectl exec -n spire $(./bin/kubectl get pod -n spire -o name) -- cat bundle.pem | openssl x509 -noout -subject)
+mintx509svid_out=mintx509svid-out.txt
```

Comment:
I'm not sure i understand the reason for this change...

Reply:
The `spire-server x509 mint` command executed in this script was writing to a directory in the container. We don't have the ability to read files from scratch containers because there are no utilities like `cat`. Rather than writing to files in the container, I opted to just parse the command output on `stdout`.

Alternatively, we could try to introduce a bind mount into the container and have the command write its output there.

Reply:
Parsing unstructured text is a little gross. How bad is the bind mount path? I'm might be ok with this for now. Presumably at some point we get --format=json for this CLI command which should improve things significantly.

Reply:
Taking another look at this test, it is deploying the containers with kind + Kubernetes. I'm not too familiar with kind, so it would be nontrivial for me at least, but there's probably a way to make it work.

Reply:
Ugg. Yeah, let's just keep this for now. We can improve it when structured output support lands for this command.